src/OpenTelemetry.Instrumentation.Wcf/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Add possibility to record exceptions from the client side.
```

Comment:
nit: Add this entry below the current one. Each section is sorted chronologically.